public/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/values.yaml

```
@@ -0,0 +1,6 @@
+# values.yaml
+podLabels:
+    azure.workload.identity/use: "true"
```

```suggestion
  azure.workload.identity/use: "true"
```
The indentation is a bit off, the rest looks great :)

---

Done.